USER_GUIDE.md

```diff
@@ -385,10 +386,10 @@ You can add something like the following to your pom.xml:
 
 In this case, running `mvn deploy` will automatically build and deploy the application to appengine.
 
-### I have a project that supports both flex and standard. How do I control which deployment to use?
+### I have a project that has both `app.yaml` and `appengine-web.xml` based config. How do I control which deployment to use?
 
-The plugin defaults to standard deployment if your project contains a webapp/WEB-INF/appengine-web.xml file. If
-your project also has an appengine/app.yaml and you wish to use flexible deployment, you may temporarily move the
+The plugin defaults to appengine-web.xml based deployment if your project contains a webapp/WEB-INF/appengine-web.xml
```

Comment:
Maybe `webapp/WEB-INF/appengine-web.xml` and `appengine/app.yaml` should use code formatting so it's easier to read?